master/propidx.html

```diff
@@ -124,19 +124,6 @@ <h1>Property Index</h1>
           <td><a href="https://www.w3.org/TR/2008/REC-CSS2-20080411/media.html#visual-media-group">visual</a></td>
           <td>yes</td>
         </tr>
-        <tr>
```

Comment:
I figured this isn't needed as it is defined as in the CSS spec now. It does limit what elements cursor can be used on, while CSS states all elements, but that is a common problem of most CSS properties not defined in SVG.

Reply:
That table includes other properties that are defined elsewhere but apply to SVG elements and have SVG presentation attributes (e.g., `display`, `font-family`), so I'd leave it there.

Reply:
I thought we planned to remove all the properties not defined in SVG in the property index, but maybe I misunderstood and all the ones that have a presentational attribute should stay?

Reply:
I think keeping all the presentation attribute properties in an appendix is reasonable.

But regardless, if we agreed to remove them, it should be in a separate PR cleaning up the entire table, not here.